core/src/components/datetime/datetime.ios.scss

```diff
@@ -83,33 +83,27 @@
    * this padding a snapping point if applied
    * on .calendar-month
    */
-  @include padding($datetime-ios-padding * 0.5, $datetime-ios-padding * 0.5, $datetime-ios-padding * 0.5, $datetime-ios-padding * 0.5);
+  @include padding($datetime-ios-padding * 0.5, $datetime-ios-padding * 0.5, 0, $datetime-ios-padding * 0.5);
 
   height: calc(100% - #{$datetime-ios-padding});
 }
 
 :host .calendar-day {
-  font-size: 20px;
-}
+  @include margin(-6.5px, null);
```

Comment:
Can you add a comment on how this value is calculated/where it comes from? I've had inconsistent behavior in the past with using fractional pixels (different browsers/devices may round up/down accordingly).

Reply:
Basically I was only comparing the datetime in one test on `main` and this `branch` and that was the margin amount needed to make them match. What I didn't realize is the datetime is a fixed height and all of the calendar days on `ios` adjust their height to fill the available space, whether that be shrinking them down or making them taller. So while this `margin` worked for the one test, it won't work for all. We made a spike ticket (FW-4595) to look into a better solution for this. I converted this back to a draft as a result.

Reply:
Resolving this as this branch has been completely redone.